integration/scripts/integration.sh

```diff
@@ -19,11 +20,15 @@ fi
 
 usage() {
     echo ""
-    echo "Usage: integration.sh [${INT_BUILD_START} | ${INT_RUN_TESTS} | ${INT_LOGS} | ${INT_STOP}]"
+    echo "Usage: integration.sh [${INT_BUILD} | ${INT_BUILD_START} | ${INT_RUN_TESTS} | ${INT_LOGS} | ${INT_STOP}]"
     echo ""
 }
 
-build_and_start_containers() {
+build_containers() {
+    bash -c "${DC} -f docker-compose.yml -f docker-compose.xpu.yml -f docker-compose.otel.yml -f docker-compose.spdk.yml -f docker-compose.pxe.yml build --parallel"
```

Comment:
I'm not going to stop this here, but I'd like to have a discussion around linting for scripts.  For example, 80 character line limits?

Reply:
agree, we should do it